style.css

```diff
@@ -1,3 +1,84 @@
+@import url('https://fonts.googleapis.com/css2?family=Comfortaa&display=swap');
+
+* {
+  font-family: 'Comfortaa', cursive;
+}
+
 body {
+  display: grid;
+  justify-content: center;
+  justify-items: center;
+  background: #f4f5f5 url("halloween.png") repeat scroll -2px -5px;
+  background-size: 10rem 10rem;
+}
+
+h1 {
+  border: 1px solid black;
+  background-color: white;
+  padding: 10px 20px;
+}
+
+#books {
+  border: 1px solid black;
+  background-color: white;
+}
+
+.book {
+  display: flex;
```

Comment:
- [ ] [OPTIONAL] Let's strive to keep the code DRY. This property:
```CSS
display: flex
```
has been used several times in the CSS. Please refactor it to a class and apply the class in the HTML file to elements that share the same display property.

### Example code
```CSS
.d-flex {
  display: flex;
  align-items: center;
}
```

Then apply this class to HTML element that needs this behavior:

```HTML
  <section class="d-flex">
```